x-pack/plugins/apm/public/components/app/service_map/empty_banner.tsx

```diff
@@ -14,11 +14,11 @@ import { useTheme } from '../../../hooks/use_theme';
 import { useApmPluginContext } from '../../../context/apm_plugin/use_apm_plugin_context';
 
 const EmptyBannerContainer = euiStyled.div`
-  margin: ${({ theme }) => theme.eui.gutterTypes.gutterSmall};
+  margin: ${({ theme }) => theme.eui.euiSizeXS};
```

Comment:
Shouldn't be this one also `euiSizeS`?

Reply:
You're totally right, this one was a brain fart on my part!